pkgs/data/icons/phinger-cursors/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, stdenvNoCC, fetchurl }:
+
+stdenvNoCC.mkDerivation rec {
```

Comment:
Maybe use fetchurl with postFetch instead

Reply:
Doesn't work, `sourceRoot` is needed since the archive contains multiple directories.